src/language/HTMLInstrumentation.js

```diff
@@ -87,6 +87,22 @@ define(function (require, exports, module) {
         return pos1 && pos2 && pos1.line === pos2.line && pos1.ch === pos2.ch;
     }
 
+    function getPositionFromTagId(editor, tagId) {
+        var marks = editor._codeMirror.getAllMarks(),
+            i,
+            markFound,
+            position;
+        
+        for (i = 0; i < marks.length; i++) {
+            if (marks[i].tagID === tagId) {
+                markFound = marks[i];
+            }
+        }
+        if (markFound) {
+            return markFound.find().from;
+        }
```

Comment:
If `markFound` can be false maybe you should add a return value, otherwise add a `console.error` or something.